pandas/io/formats/style.py

```diff
@@ -3492,6 +3502,19 @@ def from_custom_template(
         MyStyler : subclass of Styler
             Has the correct ``env``,``template_html``, ``template_html_table`` and
             ``template_html_style`` class attributes set.
+
+        Examples
+        --------
+        >>> from pandas.io.formats.style import Styler  # doctest: +SKIP
+        >>> from IPython.display import HTML  # doctest: +SKIP
+        >>> df = pd.DataFrame({"A": [1, 2]})  # doctest: +SKIP
+        >>> path = "path/to/template"  # doctest: +SKIP
+        >>> file = "template.tpl"  # doctest: +SKIP
+        >>> EasyStyler = Styler.from_custom_template(path, file)  # doctest: +SKIP
+        >>> HTML(EasyStyler(df).to_html(table_title="Another Title"))  # doctest: +SKIP
```

Comment:
I was a bit slow here but I would have commented that the IPython import and the `HTML(..)` command are unnecessary. Printing HTML and display in a Jupyter notebook would have worked regardless. These additions just obfuscate the example.

Reply:
thanks @attack68 

cool, can we address this as part of a follow-up @DeaMariaLeon please?

Reply:
it might be minimalist if the `path` and `file` are moved into the arguments without being pre-defined.

Reply:
Sure @MarcoGorelli 
Thanks for the review @attack68

Reply:
The HTML and importing IPython are also in the [User Guide](https://pandas.pydata.org/docs/dev/user_guide/style.html#:~:text=For%20convenience%2C%20we%20provide%20the%20Styler.from_custom_template%20method%20that%20does%20the%20same%20as%20the%20custom%20subclass.).  
For me it was not obvious that in a Jupyter notebook I didn't need those imports (I found out about these methods when working on the examples). If that was not obvious for me, it might not be obvious for other people. But I'm happy to do the required changes. 
I could also can add a comment.

Reply:
> The HTML and importing IPython are also in the [User Guide]

They are but are not necessary here, since these documentation examples do not render HTML.